int32 count
string[] label
int32[] box_x1
int32[] box_y1
int32[] box_x2
int32[] box_y2
